### PR TITLE
Configure Kokoro to build the 200t install package

### PR DESCRIPTION
**Pull request by HackerFoo**

### CODE REVIEW

File: .github/kokoro/install-200t.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	popd
+)
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Running installed toolchain tests"
+echo "----------------------------------------"
+echo "Skipped on single architecture package."
+# TODO adapt this test
```
Comment (litghost) [inline .github/kokoro/install-200t.sh:45]: You need to create a new test for this.

Comment (litghost) [reply .github/kokoro/install-200t.sh:45]: This class of test literally just caught a problem with https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1640, so it is delivering value.

**Review by acomodi — APPROVED** [review]
LGTM, @HackerFoo can we merge this or there is something else to be done?

**Comment by litghost** [comment]
> LGTM, @HackerFoo can we merge this or there is something else to be done?

In a previous review I requested that @HackerFoo add a test case for the A200T.

**Comment by HackerFoo** [comment]
The "Xilinx Series 7 - Vendor Tool Tests (Presubmit) " test failure is an unrelated `std::bad_alloc`, "Xilinx XC7A200T - Install (Presubmit)" passes:

```
========================================
Running installed toolchain tests
----------------------------------------
/tmpfs/src/github/symbiflow-arch-defs-presubmit-install-200t/build /tmpfs/src/github/symbiflow-arch-defs-presubmit-install-200t
Test project /tmpfs/src/github/symbiflow-arch-defs-presubmit-install-200t/build
    Start 1: binary_toolchain_test
1/1 Test #1: binary_toolchain_test ............   Passed  274.34 sec

100% tests passed, 0 tests failed out of 1

Total Test time (real) = 274.41 sec
/tmpfs/src/github/symbiflow-arch-defs-presubmit-install-200t
----------------------------------------
```

**Pull request merged**